ledger/sync/src/scp_network_state.rs

```diff
@@ -137,6 +136,37 @@ impl<ID: GenericNodeId + Send + AsRef<ResponderId> + DeserializeOwned + Serializ
                 .map(|node_id| node_id.as_ref().clone()),
         ))
     }
+
+    fn highest_block_index_on_network(&self) -> Option<BlockIndex> {
```

Comment:
Can you help me understand - is this the highest block index on the network, or the highest block index among my peers?

Reply:
It's the highest block agreed upon by a quorum set + blocking set. That's not strictly the network nor your peers (but it could be).
It's possible there could be a better name for this method. I'm open to suggestions.

Reply:
we could use the FBA state machine language -- perhaps `last_externalized_block` (or commit phase if your think that's more accurate)